jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -100,7 +100,7 @@
       </plugin-inline-select>
     </div>
 
-    <div v-if="dev_table_support && table_items.length > 0">
+    <div v-if="table_items.length > 0">

```

Comment:
is there a way to get the aperture photometry and markers tables to not automatically be listed for export? im assuming this line lets them in because an empty table or something is created when the app is, but they're empty and fail upon export

Reply:
done - let me know if it doesn't act as expected!

Reply:
works!